bookdown/01-intro.Rmd

```diff
@@ -1,75 +1,75 @@
 # Introduction and Overview
 
-The `r mlr_pkg("mlr3")` [@mlr3] package and [ecosystem](https://github.com/mlr-org/mlr3/wiki/Extension-Packages) provide a generic, object-oriented, and extensible framework for [classification](#tasks), [regression](#tasks), [survival analysis](#survival), and other machine learning tasks for the R language [@R].
+The `mlr3` [@mlr3] package and [ecosystem](https://github.com/mlr-org/mlr3/wiki/Extension-Packages) provide a generic, object-oriented, and extensible framework for [classification](#tasks), [regression](#tasks), [survival analysis](#survival), and other machine learning tasks for the R language [@R].
 We do not implement any [learners](#learners) ourselves, but provide a unified interface to many existing learners in R.
 This unified interface provides functionality to extend and combine existing [learners](#learners), intelligently select and tune the most appropriate technique for a [task](#tasks), and perform large-scale comparisons that enable meta-learning.
 Examples of this advanced functionality include [hyperparameter tuning](#tuning), [feature selection](#fs), and [ensemble construction](#fs-ensemble). [Parallelization](#parallelization) of many operations is natively supported.
 
 **Target Audience**
 
-`r mlr_pkg("mlr3")` provides a domain-specific language for machine learning in R.
+`mlr3` provides a domain-specific language for machine learning in R.
 We target both **practitioners** who want to quickly apply machine learning algorithms and **researchers** who want to implement, benchmark, and compare their new methods in a structured environment.
-The package is a complete rewrite of an earlier version of `r mlr_pkg("mlr")` that leverages many years of experience to provide a state-of-the-art system that is easy to use and extend.
+The package is a complete rewrite of an earlier version of `mlr` that leverages many years of experience to provide a state-of-the-art system that is easy to use and extend.
```

Comment:
Ok, all package links are now gone?

Reply:
Yes, see https://github.com/mlr-org/mlr3book/pull/106#discussion_r361338349

Reply:
All the `mlr_pkg()` links are back.